index.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
+const _ = require('lodash'); // Ironic, ain't it?
+const rules = require('./lib/rules/rules.json')
+
 module.exports.rules = require('./lib/rules/all');
+
+const all = Object.keys(rules);
+const compatible = Object.keys(rules).filter(key => rules[key].compatible);
+const incompatible = Object.keys(rules).filter(key => !rules[key].compatible);
+
+const ERROR = 2;
+const WARN = 1;
+
+module.exports.configs = {
+    'all': {
+      rules: _.zipObject(all, Array(all.length).fill(WARN))
```

Comment:
I need to return an object. I don't think `map()` will work, but can probably make it happen with `reduce()`. Guess I don't need Lodash!